### PR TITLE
Clarify that maximum number to keep is per feed

### PR DESCRIPTION
**Pull request by yubiuser**
Closes #https://github.com/FreshRSS/FreshRSS/issues/3374

Changes proposed in this pull request:
Within the archiving settings there is an option to set the maximum number of articles to keep. So far, this sounded like a **global** option, but it is **per feed**. I just amended the text (in the languages I feel confident) to clarify the option.

### CODE REVIEW

File: app/i18n/fr/conf.php [hunk 1]
```diff
@@ -133,7 +133,7 @@
 		'number' => 'Filtre n°%d',
 		'order_asc' => 'Afficher les articles les plus anciens en premier',
 		'order_desc' => 'Afficher les articles les plus récents en premier',
-		'search' => 'Recherche de « %s »',
+		'search' => 'Recherche de « %s »',
```
Comment (Alkarex) [inline app/i18n/fr/conf.php]: Did you change the whitespace there? In any case, does not seem related to this PR

**Comment by Alkarex** [comment]
Thanks and good idea.
Please add a `// DIRTY` comment at then end of the lines for other languages

File: app/i18n/fr/conf.php (hunk 1 above)
Comment (yubiuser) [reply app/i18n/fr/conf.php]: Not on purpose. Must have happend automatically by `vscode`. I'll take a look to see if I can undo that.

Comment (yubiuser) [reply app/i18n/fr/conf.php]: I have no idea what happend here or how to reverse it. Looking at the raw file and checking the ["hidden" characters ](https://www.soscisurvey.de/tools/view-chars.php) both versions are exactly the same.
Current `edge`: https://raw.githubusercontent.com/FreshRSS/FreshRSS/edge/app/i18n/fr/conf.php
My change: https://raw.githubusercontent.com/FreshRSS/FreshRSS/68c132bdd005d42ae92f2ee5c4f5d4e892dc7822/app/i18n/fr/conf.php

**Comment by Alkarex** [comment]
You had a copy-paste error regarding `// DIRTY`. Fixed with `make fix-all`

File: app/i18n/fr/conf.php (hunk 1 above)
Comment (Alkarex) [reply app/i18n/fr/conf.php]: No, the whitespace is not the same. We were using a non-breaking space. In any case, you can just avoid committing unrelated lines, but I would be curious to know what has changed it.
image

Comment (Alkarex) [reply app/i18n/fr/conf.php]: Fixed in https://github.com/FreshRSS/FreshRSS/pull/5458/commits/926774fdce386d156373c802d950b00593a490ff

Comment (yubiuser) [reply app/i18n/fr/conf.php]: Thanks.

**Review by Alkarex — APPROVED** [review]

**Comment by Alkarex** [comment]
Please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/edge/CREDITS.md

File: app/i18n/fr/conf.php (hunk 1 above)
Comment (yubiuser) [reply app/i18n/fr/conf.php]: > but I would be curious to know what has changed it.

I tried to reproduce it again, but was not able to do so.
